### PR TITLE
Update en.yaml file with up-to-date text

### PR DESCRIPTION
**Pull request by m-green**
This updates the en.yaml file with the up-to-date content.

I may have made some yaml / html errors - thought it was best to raise the PR now rather than try to set up docker etc to preview locally. Hope that's ok!

### CODE REVIEW

**Review by issyl0 — COMMENTED** [review]
General comment: We use typographical quotes (‘ instead of ') for apostrophes in a lot of places in this file but not everywhere, should we be consistent?

File: config/locales/en.yml [hunk 1]
```diff
@@ -239,7 +242,10 @@ en:
         custom_select_error: Select yes if your basic care needs are being met at the moment
       dietary_requirements:
         title: Do you have any special dietary requirements?
-        hint: For example, you're a vegetarian or you have a food allergy.
+        hint: |
+          <p class="govuk-hint app-hint">For example, you're a vegetarian or you have a food allergy.</p>
```
Comment (nickcolley) [inline config/locales/en.yml]: The hints will now be black test so I would recommend changing `govuk-hint app-hint` to `govuk-body` and removing the <br>

File: config/locales/en.yml [hunk 2]
```diff
@@ -239,7 +242,10 @@ en:
...
+          <br /><br />
+          <p class="govuk-inset-text">For now, food supplies will be delivered in standard packages. Check the ingredients lists carefully if you have a food allergy or follow a special diet. If you have special dietary requirements, some of the food may not meet them.</p>
```
Comment (NickColley) [inline config/locales/en.yml:247]:
```html
<div class="govuk-inset-text">
    <p class="govuk-body">...</p>
</div>
```

File: config/locales/en.yml (hunk 1 above)
Comment (nickcolley) [reply config/locales/en.yml]: actually need to double check this.

File: config/locales/en.yml (hunk 2 above)
Comment (m-green) [reply config/locales/en.yml:247]: Thanks Nick - is this correct now? https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/113/commits/388cc68e88c2bc7bc4d0d44180cad19112000ab3

File: config/locales/en.yml [hunk 3]
```diff
@@ -239,7 +242,11 @@ en:
         custom_select_error: Select yes if your basic care needs are being met at the moment
       dietary_requirements:
         title: Do you have any special dietary requirements?
-        hint: For example, you're a vegetarian or you have a food allergy.
+        hint: |
+          <p class="govuk-hint app-hint">For example, you're a vegetarian or you have a food allergy.</p>
+          <div class="govuk-inset-text">
```
Comment (NickColley) [inline config/locales/en.yml]: The component seems  wrap all this code with a div and govuk-body class which I did not expect.

So I think your previous revision was fine, sorry :man_facepalming:

**Review by NickColley — APPROVED** [review]

File: config/locales/en.yml (hunk 3 above)
Comment (m-green) [reply config/locales/en.yml]: No worries - I've reverted it

**Review by issyl0 — APPROVED** [review]
This is looking great. Thanks Mark!

**Pull request merged**